lldb/test/API/driver/terminal/TestTerminalDimensions.py

```diff
@@ -0,0 +1,22 @@
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+from lldbsuite.test.lldbpexpect import PExpectTest
+
+
+class TerminalDimensionsTest(PExpectTest):
+    NO_DEBUG_INFO_TESTCASE = True
+
+    @skipIfAsan
+    def test(self):
+        """Test that the lldb driver correctly reports the (PExpect) terminal dimension."""
+        self.launch(dimensions=(40, 40), timeout=1)
```

Comment:
Do the dimensions matter if they'll get cleared and then reset later in the test?

Reply:
They matter in the sense that they need to be different (noted in the comment below). I. need to remove the `, timeout=1` because that was just for local debugging.